### DISCUSSION TITLE
Select box does not work with initial select

### DISCUSSION THREAD

**Discussion by tonqa** [buefy/buefy#3766] [Q&A]
Hi,

Please check that the b-select element does not work with initial values. The selected value is shown for a short time and disappears then after a second. This should not be.

```html
<b-select>
  <option value="en" selected>English</option>
  <option value="de">Deutsch</option>
</b-select>
```

What do I use instead now and what works fine is the standard input:

```html
<select>
  <option value="en" selected>English</option>
  <option value="de">Deutsch</option>
</select>
```

**Comment by jtommy** [comment]
You should use v-model or value on b-select

**Comment by tonqa** [comment]
And how do I then preselect a value?

**Reply by jtommy** [reply]
How do you get the value of the select?

**Reply by tonqa** [reply]
It's a constant value, boy..

**Reply by jtommy** [reply]
I know but your code looks like you are using the "old" way to submit a form. The select contains iso codes that you are going to use.. how do you listen the select? 
Btw the only way is my first answer! (Vue binding concept)
